modules/utxo-lib/src/bitgo/parseInput.ts

```diff
@@ -30,6 +30,8 @@ export interface ParsedSignatureScript2Of3 extends ParsedSignatureScript {
        For p2sh, the placeholder is OP_0 (number 0) */
     | [Buffer | 0, Buffer | 0, Buffer | 0];
   pubScript: Buffer;
+  redeemScript: Buffer | undefined;
+  witnessScript: Buffer | undefined;
```

Comment:
Is there any reason these use `| undefined` instead of the `?` optional operator? I guess this goes along w/ the usage of `omitUndefined` below?

Reply:
this forces the property to be set explicitly, avoiding errors by accidental omission